monkestation/code/modules/antagonists/living_lube/living_lube_event.dm

```diff
@@ -0,0 +1,32 @@
+/datum/round_event_control/living_lube
+	name = "Ghost of Honks Past"
+	typepath = /datum/round_event/ghost_role/living_lube
+	weight = 2 //don't want this little lube appearing Too often
+	max_occurrences = 1
+
+/datum/round_event/ghost_role/living_lube
+	minimum_required = 1
+	role_name = "Ghost of Honks Past"
```

Comment:
Perhaps change this to "Living Lube" so ghosts know what it means when it shows up